srcpkgs/sqlite-replication/template

```diff
@@ -29,15 +28,21 @@ pre_configure() {
 	sed -i -e 's/ -ltinfo//g' configure
 }
 
+post_install() {
+	vmkdir usr/lib/pkgconfig
+	mv "$DESTDIR"/usr/lib{/sqlite-replication,}/pkgconfig/sqlite3.pc
+	# Alpine just deletes the binary, but someone may want it for debugging
+	mv "$DESTDIR"/usr/bin/sqlite3{,.sqlite-replication}
+}
+
 sqlite-replication-devel_package() {
 	depends="sqlite-replication>=${version}_${revision}"
-	replaces="sqlite-devel>=0"
-	provides="sqlite-devel-${version}_${revision}"
+	conflicts="sqlite-devel>=0"
 	short_desc+=" - development files"
 	pkg_install() {
 		vmove usr/include
```

Comment:
If we want to make -devel not conflict the header files probably need to change.

Reply:
the directory would need to change, but that is just extra work for no gain